cpp/ps0001/problem 1.cpp

```diff
@@ -1,20 +1,23 @@
 #include <iostream>
 using namespace std;
 int main(){
-    int num1;
-    int num2;
-    int num3;
-    int sum;
+    // prompting user for 3 numbers to add 1/2
+    short num1;
+    short num2;
+    short num3;
+    short sum;
     cout <<"please enter first number\n";
     cin >>num1;
     cout <<"please enter second number\n";
     cin >>num2;
     cout <<"please enter third number\n";
     cin >>num3;
+
+    // outputting the result of the summation process 3/3
     sum = (num1 + num2 + num3);
     cout <<"sum : \n";
     cout << sum;
 
-  return 0;
-
-}
\ No newline at end of file
+    return 0;
+}
+// great work 😎 8/10 ⭐

```

Comment:
- [x] program is in the correct path
- [x] no syntax errors
- [x] no bugs occurs while execution
- [x] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code